config/routes.rb

```diff
@@ -1,6 +1,5 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
```

Comment:
## [OPTIONAL]
- [ ] It would be nice if you add a root route here to prevent your app to have a default black page when the app is open. for example ` root "users#index"`

Reply:
resolved! ✔️